bvm/ballerina-profiler/src/main/java/io/ballerina/runtime/profiler/ui/HttpServer.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.runtime.profiler.ui;
+
+import io.ballerina.runtime.profiler.util.Constants;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static io.ballerina.runtime.profiler.util.Constants.OUT_STREAM;
+
+/**
+ * This class contains the HTTP server of the Ballerina profiler.
+ *
+ * @since 2201.8.0
+ */
+public class HttpServer {
+
+    public void initializeHTMLExport() throws IOException {
+        OUT_STREAM.printf(" ○ Output: " + Constants.ANSI_YELLOW
+                + "target/bin/ProfilerOutput.html" + Constants.ANSI_RESET + "%n");
```

Comment:
For single bal file this will be in the same directory. We can address these suggestions in a later PR.

Reply:
We need to pass the current directory to the `Profiler.jar` from CLI. I'll address this in https://github.com/ballerina-platform/ballerina-lang/issues/41224